custom/bihar/management/commands/bihar_cleanup_case_2.py

```diff
@@ -58,9 +58,9 @@ def handle(self, *args, **options):
                     if case.user_id == MOTECH_ID:
                         case.type = "task"
                         self.cases_to_save[case.case_id] = case
-                        print(u"Case with name " + case.name + u" updated to type " + case.type)
+                        print("Case with name " + case.name.encode('ascii', 'xmlcharrefreplace') + " updated to type " + case.type)
                     else:
-                        print(u"Type not updated for case with name " + case.name)
+                        print("Type not updated for case with name " + case.name.encode('ascii', 'xmlcharrefreplace'))
```

Comment:
the output is passed to a file when the command is run.  I'm not sure either why names are needed, but I figured it would be faster to debug this than to figure out all the context.